scripts/process.py

```diff
@@ -1,30 +1,48 @@
 import csv
-#import os
-import urllib
-#import dataconverters.xls
-sources = ["http://www3.epa.gov/climatechange/images/indicator_downloads/glaciers_fig-1.csv"]
+import requests
+
+from bs4 import BeautifulSoup
+
+source = 'https://www.epa.gov/climate-indicators/climate-change-indicators-glaciers'
 archive = 'archive/glaciers_fig-1.csv'
 data = 'data/glaciers.csv'
 
-def string_between(string, before, after):
-	temp = string.split(before)[1]
-	temp = temp.split(after)[0]
-	return temp
+def get_glaciers_data():
+    response = requests.get(source)
+    html = response.content
+    soup = BeautifulSoup(html, 'html.parser')
+    a = soup.find_all('a', href=True)
+    glacier_url = ''
+    for elem in a:
+        if 'glacier' in elem['href']:
```

Comment:
@gradedSystem I'm not sure if this is a good check...

Reply:
probably you need to add at least additional check for `.csv` format

Reply:
@anuveyatsu added regex checker  and added check for `.csv` files